spec/graphql/mutations/create_organization_mutation_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'createOrganization mutation' do
+  let!(:current_user) { create :user }
+  let(:organization_data) do
+    {
+      'name' => 'foobar',
+      'displayName' => 'Foobar',
+      'description' => 'This is the foobar',
+    }
+  end
```

Comment:
Oh yes, because the slug is generated in a before_create hook. You could call `set_slug` on the `organization` object before accessing the `values`.